native/golang/types/common/options.go

```diff
@@ -11,6 +11,10 @@
 
 package types
 
+type IoserverOptions struct {
+	NumWorkers int `json:"numWorkers,omitempty"`
```

Comment:
I might be reading this wrong, and this is probably a dumb question, but...

Since this property is optional (._.?), what should the default be?

Reply:
The default is defined in `parseOptions` as part of the `flag.Int` function call 😋 

See here:
https://github.com/zowe/zowe-native-proto/blob/737aa256ea679bcf2d9a3ed9e4e869a847ec678d/native/golang/main.go#L25-L37